internal/controller/streams.go

```diff
@@ -62,7 +44,7 @@ func ServerIsHealthy() bool {
 }
 
 // NewStream returns a new stream.
-func NewStreams(ctx context.Context, logger *zap.SugaredLogger, conn *grpc.ClientConn) (*streamManager, error) {
+func NewStreams(ctx context.Context, logger *zap.SugaredLogger, conn *grpc.ClientConn) (*streamClient, error) {
```

Comment:
Match the type name in the constructor name.
```suggestion
func NewStreamClient(ctx context.Context, logger *zap.SugaredLogger, conn *grpc.ClientConn) (*streamClient, error) {
```